org.telegram.desktop.yml

```diff
@@ -50,8 +50,11 @@ cleanup:
   - /doc
   - /include
   - /lib/cmake
+  - /lib/libexec
```

Comment:
I don't think we should cleanup `/lib/libexec` globally. It would be easy to forget about and run into issue in case of someone modules actually installs something that's needed there.

Reply:
No one would create lib/libexec, it's a custom directory